src/MOIWrapper.jl

```diff
@@ -147,174 +134,146 @@ MOI.supports(::Optimizer, ::MOI.NumberOfThreads) = false
 # ========================================
 #   Supported constraints and objectives
 # ========================================
-MOI.supports_constraint(::Optimizer,
-    ::Type{MOI.ScalarAffineFunction{Float64}}, ::Type{<:SCALAR_SETS}
-) = true
 
-MOI.supports_constraint(::Optimizer,
-    ::Type{MOI.SingleVariable}, ::Type{<:SCALAR_SETS}
-) = true
+function MOI.supports_constraint(
+    ::Optimizer, ::Type{MOI.ScalarAffineFunction{Float64}}, ::Type{S}
```

Comment:
Why creating this variable S? It's not recommended by Julia style guide : https://docs.julialang.org/en/v1/manual/style-guide/index.html#Don't-use-unnecessary-static-parameters-1